docs/deployment/teams.md

```diff
@@ -27,8 +27,6 @@ Semgrep primarily divides users into two roles:
 * `admin`
 * `member`
 
-Optionally, you can assign members to a third role, **manager**, for more granular assignment of user access. Managers are a subset of members with some additional capabilities and scopes.
```

Comment:
I removed this line because the Manager role is described in the correct scope further down, and doesn't apply to the non-Teams part of the doc, but let me know if this mention is intentional.

Reply:
Yep, it's meant as a soft intro to "what are teams and why would i use them?" I think I can add a link to the teams instead to strengthen the connection.

Reply:
I can't `suggest` on deleted lines so I will push a suggestion you can revert :)

Reply:
The new one looks good, ty!